gatsby-node.js

```diff
@@ -71,6 +71,37 @@ exports.createPages = ({ graphql, actions }) => {
         })
       })
     )
+    resolve(
+      graphql(
+        `
+          {
+            allMarkdownRemark(limit: 1000) {
+              group(field: frontmatter___author) {
+                fieldValue
```

Comment:
You may be able to combine this with your first query using [field aliases](https://graphqlmastery.com/blog/how-to-use-graphql-aliases)